EarTrumpet/DataModel/WindowsAudio/Internal/AudioDevice.cs

```diff
@@ -25,6 +25,8 @@ public class AudioDevice : BindableBase, IAudioEndpointVolumeCallback, IAudioDev
     private readonly WeakReference<IAudioDeviceManager> _deviceManager;
     private readonly string _id;
     private readonly AudioDeviceChannelCollection _channels;
+    private readonly float _deviceVolumeMindB;
```

Comment:
Nit: Let's keep the camelCase consistent

```suggestion
    private readonly float _deviceVolumeMinDb;
```

Reply:
Okay. By the way this naming is originated from the also camelCased Core Audio API function signature.